docs/docs/35-references/10-promotion-steps.md

```diff
@@ -986,6 +986,33 @@ an Argo CD `Application` to sync after previous steps have updated a remote
 branch referenced by the `Application`. This step is commonly the last step in a
 promotion process.
 
+:::note
+For Kargo `Stage`s to modify and manage Argo CD `Application` resources,
+explicit authorization must be granted in the `Application` manifests.
+This is achieved by including the `kargo.akuity.io/authorized-stage` annotation.
+
+The annotation must be formatted as follows:
+
+```yaml
+kargo.akuity.io/authorized-stage: "<project-name>:<stage-name>"
+```
```

Comment:
~~~suggestion
For an Argo CD `Application` resource to be managed by a Kargo `Stage`,
the `Application` _must_ have an annotation of the following form:

```yaml
kargo.akuity.io/authorized-stage: "<project-name>:<stage-name>"
```

Such an annotation offers proof that a user who is themselves authorized
to update the `Application` in question has consented to a specific
`Stage`updating the `Application` as well.
~~~